velox/dwio/dwrf/common/IntEncoder.h

```diff
@@ -180,6 +192,12 @@ class IntEncoder {
   }
   FOLLY_ALWAYS_INLINE void writeLongLE(int64_t val);
 
+  FOLLY_ALWAYS_INLINE void writeVuHugeInt(uint128_t val);
+  FOLLY_ALWAYS_INLINE void writeVsHugeInt(int128_t val) {
+    writeVuHugeInt(ZigZag::encodeInt128(val));
+  }
+  FOLLY_ALWAYS_INLINE void writeHugeIntLE(int128_t val);
```

Comment:
Leave an empty line in between

Reply:
Updated. And I removed 'writeHugeIntLE' method due to https://github.com/facebookincubator/velox/pull/11431#discussion_r1843709692. Thanks.